Hardware/index.md

```diff
@@ -86,3 +90,21 @@ We support PC99-style Intel Architecture Platforms.
 | [PC99 (64-bit)](IA32) | x64  | VT-X           | VT-D  | [FC (without VT-X, VT-D and fastpath)][X64] | Data61         | seL4 Foundation        |
 
 [X64]: /projects/sel4/verified-configurations.html#x64
+
+
+---
+
+## *Unmaintained platforms*
+
+
+### *ARM*
```

Comment:
I would not make them italic -- that makes them look more important. If at all we should try to de-emphasise them.

Reply:
changed everything to grey. I'm not sure I used the right way to colour text but it seems to work